Sources/WebDriver/Session.swift

```diff
@@ -317,7 +317,22 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
-    /// Deletes the current session.
+    /// - Parameter orientation: Orientation the window will flip to {LANDSCAPE|PORTRAIT}
+    public func portraitOrientation(orientation: ScreenOrientation = .portrait) throws {
+        try webDriver.send(Requests.SessionOrientation.Post(session: id, orientation: orientation))
+    }
+
+    /// - Parameter orientation: Orientation the window will flip to {LANDSCAPE|PORTRAIT}
+    public func landscapeOrientation(orientation: ScreenOrientation = .landscape) throws {
+        try webDriver.send(Requests.SessionOrientation.Post(session: id, orientation: orientation))
+    }
+
+    /// - Parameter orientation: Orientation the window is oriented to {LANDSCAPE|PORTRAIT}
+    public func orientation() throws -> String {
```

Comment:
This would be best expressed as a property returning the enum you defined:
`public var orientation: ScreenOrientation { get throws { ... } }`